glutin/src/api/android/mod.rs

```diff
@@ -57,41 +35,20 @@ impl Context {
     ) -> Result<(winit::window::Window, Self), CreationError> {
         let win = wb.build(el)?;
         let gl_attr = gl_attr.clone().map_sharing(|c| &c.0.egl_context);
-        let nwin = unsafe { android_glue::get_native_window() };
-        if nwin.is_null() {
+        let nwin = ndk_glue::native_window();
+        if nwin.is_none() {
             return Err(OsError("Android's native window is null".to_string()));
         }
```

Comment:
```suggestion
        let nwin = nwin.ok_or_else(|| OsError("Android's native window is null".to_string()))?;
```

Then you can remove the `.as_ref().unwrap()` below too.

Reply:
I had tried this too, but got this error: `move occurs because value has type `Option<NativeWindow>`, which does not implement the `Copy` trait`

and I couldn't figure out how to appease the borrow checker

Reply:
Apologies for editing this suggestion in the last minute; it compiles for me with `as_ref()` :)